clients/instance/ibm-pi-dr-location.go

```diff
@@ -0,0 +1,51 @@
+package instance
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/IBM-Cloud/power-go-client/errors"
+	"github.com/IBM-Cloud/power-go-client/helpers"
+	"github.com/IBM-Cloud/power-go-client/ibmpisession"
+	"github.com/IBM-Cloud/power-go-client/power/client/p_cloud_disaster_recovery"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+)
+
+// IBMPIDisasterRecoveryLocationClient
+type IBMPIDisasterRecoveryLocationClient struct {
+	IBMPIClient
+}
+
+// NewIBMPIDisasterRecoveryLocationClient
+func NewIBMPIDisasterRecoveryLocationClient(ctx context.Context, sess *ibmpisession.IBMPISession, cloudInstanceID string) *IBMPIDisasterRecoveryLocationClient {
+	return &IBMPIDisasterRecoveryLocationClient{
+		*NewIBMPIClient(ctx, sess, cloudInstanceID),
+	}
+}
+
+func (f *IBMPIDisasterRecoveryLocationClient) Get() (*models.DisasterRecoveryLocation, error) {
```

Comment:
Looking deep this is against the client pattern for Get() and GetAll() where we need an id for GET api. Can I ask you to provide function comments for better understanding for both functions?

Reply:
added function comments